lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +266,21 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        command = if SCCLI_SERVICES.include? service
+                    # TODO : Handle the case where user wants to pass extra
+                    # arguments to OpenShift service
+                    "sccli #{service}"
+                  else
+                    "systemctl restart #{service}"
+                  end
+
+        with_target_vms(nil, single_target: true) do |machine|
+          exit_code = machine.communicate.sudo(command)
+          unless exit_code.zero?
+            @env.ui.error "#{command} failed"
```

Comment:
@LalatenduMohanty Do you think this would make debugging easier?
Instead of exiting with 126 (blindly), let return the `exit_code` that `machine.communicate.sudo(command)` is returning.
This `exit_code` is same as the one which is returned by `sccli`. Then, we can interpret this `exit_code` in context of `sccli`.